cdap-watchdog/src/main/java/co/cask/cdap/logging/write/FileMetaDataManager.java

```diff
@@ -155,7 +155,7 @@ public Void call() throws Exception {
   }
 
   /**
-   * Scans meta data and gathers all the files older than tillTime
+   * Scans meta data and gathers all the files older than untilTime
```

Comment:
looks like this comment is inaccurate. This method scans up to a limited number of records, not until a time limit.